app/main.py

```diff
@@ -1,3 +1,52 @@
+from __future__ import annotations
+
+
 class Distance:
-    # Write your code here
-    pass
+
+    def __init__(self, km: int) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: Distance | int) -> Distance:
+        return Distance(
+            self.km + other.km
+            if isinstance(other, Distance)
+            else self.km + other)
```

Comment:
Fix formatting: brackets must be located in one of two options: open and start with the text or have a line break between the text

Reply:
fixed